tee-worker/litentry/core/data-providers/src/lib.rs

```diff
@@ -211,18 +211,18 @@ impl DataProviderConfig {
 			discord_auth_token: "".to_string(),
 			achainable_url: "https://label-production.graph.tdf-labs.io/".to_string(),
 			achainable_auth_key: "".to_string(),
-			credential_endpoint: "wss://tee-staging.litentry.io".to_string(),
+			credential_endpoint: "wss://rpc.rococo-parachain.litentry.io".to_string(),
```

Comment:
This value will vary across environments but we cannot let it to be overriden by environment variable. I think we will need a compilation toggle for that.

Let's address it in another PR.

Reply:
Yes. That would be a better approach.